web/templates/shared/form.html.eex

```diff
@@ -0,0 +1,19 @@
+<%= form_for @changeset, post_path(@conn, :create), fn f -> %>
```

Comment:
I guess we are thinking this form could be used for things other than posts and thats why its in shared? I'm on board with that.  Maybe we should consider naming it something less generic than `form` if its in the shared folder though? Something like `title_body_form.eex` or something? Thoughts?